contracts/protocols/mainnet/compound_iii/helpers.sol

```diff
@@ -0,0 +1,369 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+import "./interfaces.sol";
+import { DSMath } from "../../../utils/dsmath.sol";
+
+contract CompoundIIIHelpers is DSMath {
+    /**
+     *@dev Returns ethereum address
+     */
+    function getEthAddr() internal pure returns (address) {
+        return 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+    }
+
+    function getCometRewardsAddress() internal pure returns (address) {
+        return 0x1B0e765F6224C21223AeA2af16c1C46E38885a40;
+    }
+
+    function getConfiguratorAddress() internal pure returns (address) {
+        return 0xcFC1fA6b7ca982176529899D99af6473aD80DF4F;
+    }
+
+    struct BaseAssetInfo {
+        address token;
+        address priceFeed;
+        uint256 price;
+        uint8 decimals;
+        uint64 mantissa;
+        uint64 indexScale;
+        uint64 trackingSupplyIndex;
+        uint64 trackingBorrowIndex;
+    }
+
+    struct Scales {
+        //liquidation factor, borrow factor scale
+        uint64 factorScale;
+        //scale for USD prices
+        uint64 priceScale;
+        //scale for rewards APR
+        uint64 trackingIndexScale;
+    }
+
+    struct Token {
```

Comment:
@Richa-iitr Can you also return token decimals

Reply:
I am returning token scale which is 10**decimals, wouldn't this suffice?

Reply:
Okay, but we need decimals as well.

Reply:
Okay

Reply:
done.